src/Microsoft.Extensions.DiagnosticAdapter/DiagnosticSourceAdapter.cs

```diff
@@ -42,7 +42,7 @@ private static Listener EnlistTarget(object target, Func<string, bool> isEnabled
             var methodInfos = typeInfo.DeclaredMethods;
             foreach (var methodInfo in methodInfos)
             {
-                var notificationNameAttribute = methodInfo.GetCustomAttribute<TelemetryNameAttribute>();
+                var notificationNameAttribute = methodInfo.GetCustomAttribute<DiagnosticNameAttribute>();
```

Comment:
EventNotification was the original name of this package. If there are any remaining traces of it, then it's two renames out of date.